src/Identity/test/Shared/MockHelpers.cs

```diff
@@ -67,9 +67,13 @@ public static ILookupNormalizer MockLookupNormalizer()
         var normalizerFunc = new Func<string, string>(i =>
         {
             if (i == null)
```

Comment:
Unrelated to this PR, but should we consider enforcing `is null` instead of `== null`.

Reply:
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0041. I think we should bring it up as part of a DOI thing first though.

Reply:
https://github.com/dotnet/aspnetcore/issues/39323